README.md

```diff
@@ -1,5 +1,9 @@
 # Canary Anomaly Detection
 
+This is a research prototype built as part of an internship project. 
+See the bug [1475933](https://bugzilla.mozilla.org/showdependencytree.cgi?id=1475933&hide_resolved=1) 
```

Comment:
nit: let's drop the `&hide_resolved=1` so that the full depednency tree shows up